test/any/web_apis/web_apis-integration.es6.js

```diff
@@ -16,13 +16,13 @@ describe('WebAPI and api extractor', function() {
     let edge14 = global.DATA.edge14;
     let safari602 = global.DATA.safari602;
     let og = global.ObjectGraph;
-    let extractor = org.chromium.apis.web.apiExtractor.create({});
-    let apiImporter = org.chromium.apis.web.ApiImporter.create();
+    let extractor = org.chromium.apis.web.apiExtractor.create();
+    let apiImporter = org.chromium.apis.web.ApiImporter.create(null, extractor);
     let apiMatrix = org.chromium.apis.web.ApiMatrix.create({
       browserApiDAO: apiImporter.browserApiDAO,
       browserDAO: apiImporter.browserDAO,
       interfaceDAO: apiImporter.interfaceDAO,
-    });
+    }, apiImporter);
```

Comment:
Do tests pass *without* this change? If so, let's drop this. There are no realistic scenarios where an `apiMatrix` runs on the same machine as an `apiImporter`.